### DISCUSSION TITLE
GitOps Engine

### DISCUSSION THREAD

**Discussion by tobru** [appuio/appuio-cloud-community#17] [Ideas]
Provide an integrated way to the APPUiO Cloud user to deploy workload to the cluster via GitOps.

**Comment by jkleinlercher** [comment]
Is there already some progress on this topic? We also highly need that and since gitops becomes the state of the art to manage your apps and infra we would love to do that with appuio!
Also integration with Crossplane would fit in this direction!

**Reply by tobru** [reply]
Would you mind sharing your ideas and use-cases at https://roadmap.appuio.cloud? That would help us a lot in understanding your needs and prioritize accordingly.
